libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -57,15 +63,15 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: TYPE
     // @DisplayName: GPS type
     // @Description: GPS type
-    // @Values: 0:None,1:AUTO,2:uBlox,3:MTK,4:MTK19,5:NMEA,6:SiRF,7:HIL,8:SwiftNav,9:PX4-UAVCAN,10:SBF,11:GSOF,12:QURT,13:ERB,14:MAV,15:NOVA
+    // @Values: 0:None,1:AUTO,2:uBlox,3:MTK,4:MTK19,5:NMEA,6:SiRF,7:HIL,8:SwiftNav,9:PX4-UAVCAN,10:SBF,11:GSOF,12:QURT,13:ERB,14:MAV,15:NOVA,16:UAVCAN
```

Comment:
I'd prefer we didn't go playing with parameters as anyone using CAN with a master build after this goes in will then have a bad parameter once this is changed back to 9. Anyone using master should know that parameters can change without any protection though so I'm not going to push too much on this.